setup.py

```diff
@@ -19,31 +19,45 @@ def _post():
     nltk.download('stopwords')
     nltk.download('punkt')
 
+
 class PostInstallCommand(install):
     """Post-installation for installation mode."""
     def run(self):
         install.run(self)
         _post()
 
+
 class PostDevelopCommand(develop):
     """Post-installation for development mode."""
     def run(self):
         develop.run(self)
         _post()
 
 
+# pypi does not render markdown.
+# https://stackoverflow.com/a/26737672/19741
+try:
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
+except(IOError, ImportError):
+    long_description = open('README.md').read()
```

Comment:
Is there any use of `long_description` except on pypi? 

My hope is that when we do releases, that we verify the landing pages on pypi look correct, so I was thinking the scope of impact was only to the release manager